docs/index.md

```diff
@@ -5,24 +5,27 @@ title: Fleet SDK
 titleTemplate: Fluent Ergo Toolset
 
 hero:
-  name: Fleet SDK
-  text: Fluent Ergo Toolset
-  tagline: Simple, powerful, and extensible. Meet the next generation of Ergo Platform's off-chain code SDKs.
-  actions:
-    - theme: brand
-      text: Get Started
-      link: /guide/getting-started
-    - theme: alt
-      text: View on GitHub
-      link: https://github.com/fleet-sdk
+name: Fleet SDK
+text: Fluent Ergo Toolset
+tagline: Simple, powerful, and extensible. Meet the next generation of Ergo Platform's off-chain code SDKs.
+actions:
+- theme: brand
+text: Introduction
+link: /guide/introduction
+- theme: brand
+text: Get Started
+link: /guide/getting-started
+- theme: alt
+text: View on GitHub
+link: https://github.com/fleet-sdk
 
 features:
-  - title: Simplicity first
-    details: Fleet abstracts all complex tasks and makes heavy use Fluent APIs to deliver a great developer experience.
-  - title: Power of Ergo meets JavaScript
-    details: JavaScript allows running off-chain code directly on user's browser, which means better decentralization as it makes backends unnecessary for most cases.
-  - title: "Pure JavaScript"
-    details: Enjoy the experience of writing frictionless off-chain code with a fully typed and pure JS library.
-  - title: "Write once, run anywhere."
-    details: Out-of-the-box compatible with Node.js and Browser environments.
+- title: Simplicity first
+  details: Fleet abstracts all complex tasks and makes heavy use Fluent APIs to deliver a great developer experience.
+- title: Power of Ergo meets JavaScript
+  details: JavaScript allows running off-chain code directly on user's browser, which means better decentralization as it makes backends unnecessary for most cases.
+- title: "Pure JavaScript"
+  details: Enjoy the experience of writing frictionless off-chain code with a fully typed and pure JS library.
+- title: "Write once, run anywhere."
+  details: Out-of-the-box compatible with Node.js and Browser environments.
```

Comment:
It looks like your editor messed with `index.md` indentation, can you please check what's possibly wrong?

Reply:
Yeah, IntelliJ is new to me. I'll fix this and the others tomorrow.

Reply:
You can try using Visual Studio Code as well :)